src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/RuleBasedTests.cs

```diff
@@ -192,7 +233,12 @@ public async Task CheckOnePatient(string templateName, string samplePath, DataTy
         {
             var result = await ConvertData(templateName, samplePath, dataType);
             var patients = result.SelectTokens("$.entry[?(@.resource.resourceType == 'Patient')].resource.id");
-            Assert.Equal(1, patients?.Count());
+            if (templateName == "ADT_A40" || templateName == "ADT_A41" ) {
```

Comment:
Just want to double confirm: are we going to convert these two types of messages to FHIR without a Patient resource?

Reply:
Yes @BoyaWu10 , we are not creating the Patient resource in ADT_A40 and ADT_A41. We are creating the Linkage resource in these messages.